modules/actor/actor-wfrp4e.js

```diff
@@ -918,6 +929,11 @@ export default class ActorWfrp4e extends Actor {
         testData.testDifficulty = game.wfrp4e.config.difficultyModifiers[html.find('[name="testDifficulty"]').val()];
         testData.successBonus = Number(html.find('[name="successBonus"]').val());
         testData.slBonus = Number(html.find('[name="slBonus"]').val());
+        if (testData.unofficialGrimoire) {
+          testData.ingredientMode = html.find('[name="ingredientTypeSelected"]').val();
+          testData.overchannelling = Number(html.find('[name="overchannelling"]').val());
+          testData.quickcasting = html.find('[name="quickcasting"]').is(':checked');
+        }
```

Comment:
Can you add  `game.wfrp4e.utility.logHomebrew("unofficialgrimoire")`

Reply:
done